lib/widgets/detail_views/sliver_logo.dart

```diff
@@ -21,42 +21,41 @@ class SliverLogo extends StatelessWidget {
   final String? logoDirectusUrl;
   final BoxFit boxfit;
   final bool noShimmeringLoading;
+
   @override
   Widget build(BuildContext context) {
     return Align(
       alignment: Alignment.bottomCenter,
       child: SizedBox.square(
-        child: ListView(
+        child: SingleChildScrollView(
```

Comment:
why `SingleChildScrollView` here?

Reply:
This was changed, because in scientific club view ListView has tottally "hidden" the image behind it not allowing you to click on it. It turned out that SingleChildScrollView takes less space and thus allowng you to click on that image behind.